src/OrbitBase/ThreadPool.cpp

```diff
@@ -19,16 +20,27 @@
 namespace orbit_base {
 namespace {
 
+ABSL_CONST_INIT absl::Mutex g_default_thread_pool_mutex(absl::kConstInit);
+ABSL_CONST_INIT std::shared_ptr<ThreadPool> g_default_thread_pool
```

Comment:
The reason is that our interface for creating a ThreadPool returns a std::shared_ptr<ThreadPool>, we should probably change that to a std::unique_ptr actually since we can convert a std::unique_ptr to a std::shared_ptr, but not the other way around.

Reply:
That's my fault. We need that shared_ptr for `TrySchedule` in `ScopedStatus` but it should really be an implementation detail of `ThreadPool` - but that's non trivial. I might be able to tackle that in the fixit week.

Reply:
Okay, let's leave it as is for now and clean it up (or not - it's not a huge deal after all).